Examples/Scripts/Python/material_recording.py

```diff
@@ -70,7 +70,7 @@ def runMaterialRecording(
         detectorConstructionFactory=detectorConstructionFactory,
         randomNumbers=rnd,
         inputParticles=evGen.config.outputParticles,
-        outputMaterialTracks="material_tracks",
+        outputMaterialTracks="material-tracks",
```

Comment:
```suggestion
        outputMaterialTracks="material_tracks",
```

Reply:
I had that initially, but then I need to change it everywhere (including the python tests), so the initially short PR gets more complicated.

I will address that consistently in a separate PR.

Reply:
sure no requirement but this change in this pr here no?